packages/react-components/src/components/chart/baseChart.tsx

```diff
@@ -148,6 +139,14 @@ const BaseChart = ({
     },
   });
 
+  // removed from the chart configuration because it will be refactored into echarts in an upcoming PR
```

Comment:
nit: probably worth moving into the useTrendcursors hook so all the change wrt TC lives there

Reply:
👍